python/pyspark/sql/streaming/__init__.py

```diff
@@ -19,3 +19,4 @@
 from pyspark.sql.streaming.readwriter import DataStreamReader, DataStreamWriter  # noqa: F401
 from pyspark.sql.streaming.listener import StreamingQueryListener  # noqa: F401
 from pyspark.errors import StreamingQueryException  # noqa: F401
+from pyspark.sql.streaming.stateful_processor import StatefulProcessor, StatefulProcessorHandle # noqa: F401
```

Comment:
Where is this change used ?

Reply:
We need this, otherwise we cannot import StatefulProcessor and StatefulProcessorHandle
```
>>> from pyspark.sql.streaming import StatefulProcessor, StatefulProcessorHandle
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'StatefulProcessor' from 'pyspark.sql.streaming' (/Users/bo.gao/workplace/spark/python/pyspark/sql/streaming/__init__.py)
```